.github/workflows/generate-phapi-synthetic-data-spreadsheet.yaml

```diff
@@ -44,20 +44,35 @@ jobs:
           ref: qa
           fetch-depth: 0
           path: synthetic-records
-      - name: Generate Spreadsheet
+      - name: Generate PHAPI Spreadsheet
         env:
           GITHUB_TOKEN_USERNAME: ${{ secrets.BUILD_GITHUB_USERNAME }}
           GITHUB_TOKEN: ${{ secrets.BUILD_GITHUB_TOKEN }}
         run: |
           export LIGHTHOUSE_SETTINGS=${GITHUB_WORKSPACE}/consumer-docs/.github/build/settings.xml
-          echo "Building spreadsheet..."
+          echo "Building PHAPI spreadsheet..."
           ${GITHUB_WORKSPACE}/synthetic-records/minimart-manager/summary-xlsx.sh
           mkdir -p ${GITHUB_WORKSPACE}/consumer-docs/patient-health-v0
           mv ${GITHUB_WORKSPACE}/synthetic-records/health-test-patient-data.xlsx \
             ${GITHUB_WORKSPACE}/consumer-docs/patient-health-v0/health-test-patient-data.xlsx
           cd ${GITHUB_WORKSPACE}/consumer-docs
           git add patient-health-v0/health-test-patient-data.xlsx
-      - name: Upload Spreadsheet
+      - name: Generate H3CAPI Spreadsheet
+        env:
+          GITHUB_TOKEN_USERNAME: ${{ secrets.BUILD_GITHUB_USERNAME }}
+          GITHUB_TOKEN: ${{ secrets.BUILD_GITHUB_TOKEN }}
+        run: |
+          export LIGHTHOUSE_SETTINGS=${GITHUB_WORKSPACE}/consumer-docs/.github/build/settings.xml
```

Comment:
Create an `Initialize` step and save this to `GITHUB_ENV`. No point in duplicating it.